### PR TITLE
Fix minor erros in Helm chart

### PR DESCRIPTION
**Pull request by warrenbailey**
# Motivation and Context
Fixing minor changes in helm chart ready for preprod

# What has changed

# How to test?

# Links

# Screenshots (if appropriate):

### CODE REVIEW

File: _infra/helm/templates/deployment.yaml [hunk 1]
```diff
@@ -6,30 +6,30 @@ spec:
...
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
           ports:
             - name: http-server
-              containerPort: 8087
+              containerPort: {{ .Values.service.port }}
           readinessProbe:
             httpGet:
               path: /info
-              port: 8087
+              port: {{ .Values.service.port }}
```
Comment (jterryons) [inline _infra/helm/templates/deployment.yaml]: Would it be worth changing the config name here? this isnt the service port, its the container port. Might confuse us later on.

Comment (warrenbailey) [reply _infra/helm/templates/deployment.yaml]: yep good spot

File: _infra/helm/frontstage/values.yaml [hunk 2]
```diff
@@ -1,11 +1,23 @@
 replicaCount: 1
 
 namespace: minikube
+env: minikube
+
+container:
+  port: 80
```
Comment (jterryons) [inline _infra/helm/frontstage/values.yaml]: 8087

File: _infra/helm/frontstage/values.yaml [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
 
 service:
   port: 8087
```
Comment (jterryons) [inline _infra/helm/frontstage/values.yaml]: 80

**Pull request merged**